<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Irynas-MacBook-Pro.local" ignored="1" name="calculator base" tests="38" failures="10" timestamp="11 Aug 2019 08:53:10 GMT" time="0.048" errors="0">
  <testcase name="sumDoubleTest" time="0.002" classname="testng.SumDoubleTest"/>
  <testcase name="subDoubleTest" time="0.002" classname="testng.SubDoubleTest"/>
  <testcase name="subDoubleTest" time="0.0" classname="testng.SubDoubleTest"/>
  <testcase name="sumDoubleTest" time="0.0" classname="testng.SumDoubleTest"/>
  <testcase name="sumDoubleTest" time="0.0" classname="testng.SumDoubleTest"/>
  <testcase name="subDoubleTest" time="0.0" classname="testng.SubDoubleTest"/>
  <testcase name="sumDoubleTest" time="0.0" classname="testng.SumDoubleTest"/>
  <testcase name="subDoubleTest" time="0.001" classname="testng.SubDoubleTest"/>
  <testcase name="sumDoubleTest" time="0.0" classname="testng.SumDoubleTest"/>
  <testcase name="sumDoubleTest" time="0.0" classname="testng.SumDoubleTest"/>
  <testcase name="subDoubleTest" time="0.0" classname="testng.SubDoubleTest"/>
  <testcase name="tgTest" time="0.001" classname="testng.TgTest">
    <failure type="java.lang.AssertionError" message="wrong result. Method implementation is wrong expected [0.0] but found [NaN]">
      <![CDATA[java.lang.AssertionError: wrong result. Method implementation is wrong expected [0.0] but found [NaN]
at testng.TgTest.tgTest(TgTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- tgTest -->
  <testcase name="multDoubleTest" time="0.001" classname="testng.MultDoubleTest">
    <failure type="java.lang.AssertionError" message="Operation error: result was rounded! expected [2.31] but found [2.0]">
      <![CDATA[java.lang.AssertionError: Operation error: result was rounded! expected [2.31] but found [2.0]
at testng.MultDoubleTest.multDoubleTest(MultDoubleTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- multDoubleTest -->
  <testcase name="multDoubleTest" time="0.0" classname="testng.MultDoubleTest"/>
  <testcase name="tgTest" time="0.0" classname="testng.TgTest">
    <failure type="java.lang.AssertionError" message="wrong result. Method implementation is wrong expected [0.0699268] but found [1.0]">
      <![CDATA[java.lang.AssertionError: wrong result. Method implementation is wrong expected [0.0699268] but found [1.0]
at testng.TgTest.tgTest(TgTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- tgTest -->
  <testcase name="multDoubleTest" time="0.0" classname="testng.MultDoubleTest"/>
  <testcase name="tgTest" time="0.0" classname="testng.TgTest">
    <failure type="java.lang.AssertionError" message="wrong result. Method implementation is wrong expected [0.1095178] but found [1.0]">
      <![CDATA[java.lang.AssertionError: wrong result. Method implementation is wrong expected [0.1095178] but found [1.0]
at testng.TgTest.tgTest(TgTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- tgTest -->
  <testcase name="sqrtTest" time="0.001" classname="testng.SqrtTest"/>
  <testcase name="multDoubleTest" time="0.0" classname="testng.MultDoubleTest"/>
  <testcase name="multDoubleTest" time="0.0" classname="testng.MultDoubleTest"/>
  <testcase name="sqrtTest" time="0.0" classname="testng.SqrtTest"/>
  <testcase name="sqrtTest" time="0.0" classname="testng.SqrtTest"/>
  <testcase name="powDoubleTest" time="0.0" classname="testng.PowTest"/>
  <testcase name="divDoubleTest" time="0.0" classname="testng.DivDoubleTest"/>
  <testcase name="powDoubleTest" time="0.0" classname="testng.PowTest"/>
  <testcase name="divDoubleTest" time="0.0" classname="testng.DivDoubleTest"/>
  <testcase name="powDoubleTest" time="0.001" classname="testng.PowTest"/>
  <testcase name="divDoubleTest" time="0.0" classname="testng.DivDoubleTest"/>
  <testcase name="powDoubleTest" time="0.0" classname="testng.PowTest"/>
  <testcase name="divDoubleTest" time="0.0" classname="testng.DivDoubleTest"/>
  <testcase name="powDoubleTest" time="0.0" classname="testng.PowTest">
    <failure type="java.lang.AssertionError" message="The result is rounded! Error in method implementation! expected [2.14354693] but found [2.0]">
      <![CDATA[java.lang.AssertionError: The result is rounded! Error in method implementation! expected [2.14354693] but found [2.0]
at testng.PowTest.powDoubleTest(PowTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- powDoubleTest -->
  <testcase name="divideByZero" time="0.0" classname="testng.DivDoubleTest"/>
  <testcase name="cgTest" time="0.0" classname="testng.CtgTest"/>
  <testcase name="cosTest" time="0.0" classname="testng.CosTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.0]
at testng.CosTest.cosTest(CosTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
  <testcase name="cgTest" time="0.0" classname="testng.CtgTest">
    <failure type="java.lang.AssertionError" message="wrong result. Method implementation is wrong expected [0.0699268] but found [0.7615941559557649]">
      <![CDATA[java.lang.AssertionError: wrong result. Method implementation is wrong expected [0.0699268] but found [0.7615941559557649]
at testng.CtgTest.cgTest(CtgTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- cgTest -->
  <testcase name="cosTest" time="0.0" classname="testng.CosTest">
    <failure type="java.lang.AssertionError" message="expected [0.0] but found [0.8414709848078965]">
      <![CDATA[java.lang.AssertionError: expected [0.0] but found [0.8414709848078965]
at testng.CosTest.cosTest(CosTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
  <testcase name="cgTest" time="0.0" classname="testng.CtgTest">
    <failure type="java.lang.AssertionError" message="wrong result. Method implementation is wrong expected [0.1095178] but found [0.9999925467214317]">
      <![CDATA[java.lang.AssertionError: wrong result. Method implementation is wrong expected [0.1095178] but found [0.9999925467214317]
at testng.CtgTest.cgTest(CtgTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- cgTest -->
  <testcase name="cosTest" time="0.0" classname="testng.CosTest">
    <failure type="java.lang.AssertionError" message="expected [0.28366218546] but found [0.9589242746631385]">
      <![CDATA[java.lang.AssertionError: expected [0.28366218546] but found [0.9589242746631385]
at testng.CosTest.cosTest(CosTest.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
  <testcase name="isPositive" time="0.0" classname="testng.IsPositiveTest">
    <ignored/>
  </testcase> <!-- isPositive -->
</testsuite> <!-- calculator base -->
